arm/vrb_test.c

```diff
@@ -0,0 +1,76 @@
+#include <inttypes.h>
```

Comment:
YAY TESTS

Reply:
Yeah, I was really doubting my mmap magic would work at all. I had to see it with my own eyes. And with a test.